simple-test-cases/openmp_section1.c

```diff
@@ -0,0 +1,29 @@
+///TAFFO_TEST_ARGS -fopenmp
+#include <stdio.h>
+
+#define N (100)
+
+int main(int argc, char *argv[]) {
+  float foo __attribute((annotate("target('foo') scalar(range(0,10) final)"))) =
+      1.0f;
+  float pop
+      __attribute((annotate("target('pop') scalar(range(0,2000) final)"))) =
+          363;
+  float result
+      __attribute((annotate("target('result') scalar(range(0,2000)) final)"))) =
+          0;
+
+#pragma omp parallel num_threads(4)
+  {
```

Comment:
Is it possible to have the open brace on the same line as the pragma?

Reply:
It should not be possible: the main reason is that the pragma is read from the pre-processor, so the compiler doesn't see the open brace.

The following warning is issued:
`./openmp_task1.c:16:37: warning: extra tokens at the end of '#pragma omp parallel' are ignored [-Wextra-tokens]`

And therefore the open brace is ignored